connect/runtime/src/main/java/org/apache/kafka/connect/runtime/AbstractHerder.java

```diff
@@ -345,8 +345,12 @@ protected Connector getConnector(String connType) {
         }
     }
 
-    public ConnectorType connectorType(String connType) {
-        return ConnectorType.from(getConnector(connType).getClass());
+    /*
+     * Retrieves ConnectorType for the corresponding connector class
+     * @param connClass class of the connector
+     */
+    public ConnectorType connectorTypeForClass(String connClass) {
+        return ConnectorType.from(getConnector(connClass).getClass());
```

Comment:
Can you point out where this method is used in a unit test, other than when the method is mocked?

Reply:
Looks like AbstractHerderTest now exercises this